packages/pytest-simcore/src/pytest_simcore/docker.py

```diff
@@ -20,12 +20,16 @@ async def async_docker_client() -> AsyncIterator[aiodocker.Docker]:
 
 
 @contextlib.asynccontextmanager
-async def _pause_container(
+async def _pause_docker_container_in_context(
```

Comment:
Q: what does "in_context" mean. this is a context manager already no need to add this suffix and the static typer and VScode shows it already

Reply:
The docker is paused only in the context this manager spans. I thought it was more readable. 
In addition think it is good to stress that these functions return a context manager. Sometimes libraries even add `_ctx` suffix to distinguish them from normal functions.

Reply:
ok. I turn it back to `paused_container`